tests/jvm/src/test/resources/metac-metacp.diff

```diff
@@ -1078,122 +1054,147 @@ flags.p.package.S#[T]
 -  tpe {
 -    typeRef {
 -      prefix {
--      symbol: "scala.specialized#"
+-      symbol: "scala/specialized#"
 -}
  accessibility {
    tag: PUBLIC
    symbol: ""
 
 
+======================
+types/Test.C#Either().
+======================
+--- metac
++++ metacp
+       singleType {
+         prefix {
+-        symbol: "scala/util/Either."
```

Comment:
This is the only regression I can see, and the issue is already tracked in https://github.com/scalameta/scalameta/issues/1625#issuecomment-402174890 I will fix that issue later this week.

Reply:
Turns out #1625 was unrelated after all, this regression has been fixed now.